client/i18n/locales/chinese/translations.json

```diff
@@ -468,5 +468,13 @@
     "invalid-protocol": "URL 必须以 http 或 https 开头",
     "url-not-image": "URL 必须直接链接到图片文件",
     "use-valid-url": "请使用有效的 URL"
+  },
+  "certification": {
+    "certifies": "Chinese: This certifies that",
```

Comment:
Should I create a PR to remove the "Chinese: " bit?  Unless we're sure we're going to pull down the translations it would look better to just be the original English.

Reply:
I would hope to include this in the soft-launch, otherwise we have contributors go through the Responsive Web Design projects in Chinese and get a certificate in English 😩 

But yes, these prefixes can be removed - I included them in the PR solely to facilitate testing for the review process.

Reply:
Agreed.  I'll create the PR just in case and add the files to the learn UI project, too.